tests/concurrent-restore/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+#
+# Copyright 2019 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+# Populate the mydumper source
+DBPATH="$TEST_DIR/restore.mydump"
+TABLE_COUNT=20
```

Comment:
Is the test reproducible using less number of tables and a static data source?

Reply:
the more tables, the higher probability to reproduce this bug. I tested four tables seems enough to reproduce, I changed the table count to 8